app/build.gradle

```
@@ -42,4 +42,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.1"
+
+    implementation "androidx.activity:activity-ktx:1.6.1"
```

```suggestion
    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.6.1'
    implementation 'androidx.activity:activity-ktx:1.6.1'
```
중간에 변수로 치환을 할 일이 없다면 `''` 스트링을 사용하는 편이 좋습니다.

---

변수를 할당하는 경우에만 ""를 사용하는 건가요?

---

엄격히 이야기하자면, gradle은 '' 와 "" 의 자료형이 다릅니다.
중간에 변수를 이용한 문자열 치환이 없다면, `''`로 하는것이 일반적입니다. (shell 스크립트와 유사함)

---

Reference:
https://phoenixnap.com/kb/bash-single-vs-double-quotes
https://groovy-lang.org/syntax.html#_single_quoted_string

---

버전관리를 따로 선언해서 하는 경우에만 ""를 쓰는게 좋겠군요